docs/tutorial.rst

```diff
@@ -1081,15 +1142,28 @@ the index of the new mutation type in the contig's list of mutation types.
 
 .. code-block:: python
 
```

Comment:
warning this API for sweeps isn't final? also do we have an issue to keep track of this?

Reply:
I think so: #1082 #1085 #1053